Lombiq.Hosting.Tenants.FeaturesGuard/Startup.cs

```diff
@@ -1,31 +1,14 @@
 using Lombiq.Hosting.Tenants.FeaturesGuard.Constants;
 using Lombiq.Hosting.Tenants.FeaturesGuard.Handlers;
-using Lombiq.Hosting.Tenants.FeaturesGuard.Models;
-using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using OrchardCore.Environment.Shell;
-using OrchardCore.Environment.Shell.Configuration;
-using OrchardCore.Environment.Shell.Descriptor.Models;
 using OrchardCore.Modules;
 
 namespace Lombiq.Hosting.Tenants.FeaturesGuard;
 
 [Feature(FeatureNames.FeaturesGuard)]
 public class Startup : StartupBase
 {
-    private readonly IShellConfiguration _shellConfiguration;
-
-    public Startup(IShellConfiguration shellConfiguration) => _shellConfiguration = shellConfiguration;
-
-    public override void ConfigureServices(IServiceCollection services)
-    {
-        services.Configure<AlwaysEnabledFeaturesOptions>(options =>
-            _shellConfiguration
-                .GetSection("Lombiq_Hosting_Tenants_FeaturesGuard:AlwaysEnabledFeaturesOptions")
-                .Bind(options));
-
+    public override void ConfigureServices(IServiceCollection services) =>
```

Comment:
Both `OrchardCoreBuilderExtensions` methods should rather be called here.

Reply:
Akin to _IdleTenantManagement_'s config, without `OrchardCoreBuilder`?
![image](https://user-images.githubusercontent.com/51411356/199211005-12228178-ad1a-45fa-bbbf-6944ea9b23fd.png)


Or is there a way to use the `OrchardCoreBuilderExtension` methods inside a _Startup_ class too? `services.AddOrchardCore()` seems to result in an exception.
![image](https://user-images.githubusercontent.com/51411356/199211185-b05bbe28-30f3-40aa-a2a5-a845ba9d6573.png)

Reply:
The problem I'm encountering now is during UI test runs, `_shellConfiguration.IsUITesting();` returns `false` in FeaturesGuard's _Startup_, while the same method returns true in _Program_.